src/views/partials/labels.handlebars

```diff
@@ -0,0 +1,18 @@
+<h3>Labels</h3>
+{{!-- List Labels --}}
+<form>
+    {{#each label}}
+        <input type="checkbox" checked id="card_label_{{@index}}">
+        <label for="card_label_{{@index}}"><p class="card-label-highlight" style="background-color: {{this.color}};"></p></label>
+    {{/each}}
+    <button type="submit" value="submit">Set Changes</button>
+</form>
+
+{{!-- Add Label --}}
+<form id="addLabel">
```

Comment:
this should all be in one form

Reply:
I put the add label form inside the other one